webextensions/manifest/background.json

```
@@ -79,8 +79,7 @@
                 "version_added": true
               },
               "safari": {
-                "notes": "Only persistent pages are supported.",
-                "partial_implementation": true,
+                "notes": "Support for non-persistent pages was added in 14.1.",
```

It doesn't always happen, but I strive to have notes framed such that the note is about the exceptional or historic case, not the general case, so notes about historic behavior recede from importance.

```suggestion
                "notes": "Before 14.1, only persistent pages are supported",
```

---

Addressed in 90183cb.